packages/config/config/devices/0x0039/39348-zw4008.json

```
@@ -73,14 +76,22 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Any button",
+					"label": "Press any button on the device",
 					"value": 0
 				},
 				{
-					"label": "2x ON button, 2x OFF button",
+					"label": "Press 2 times ON button and then 2 times OFF button, LED will flash 5 times if done successful",
```

Thats a bit long for a drop down or similar
```suggestion
					"label": "Press ON button 2x, then OFF button 2x",
```